examples/consul-ami/README.md

```diff
@@ -73,10 +87,23 @@ Your code should look more like this:
     "type": "shell",
     "inline": [
       "git clone --branch <MODULE_VERSION> https://github.com/hashicorp/terraform-aws-consul.git /tmp/terraform-aws-consul",
-      "/tmp/terraform-aws-consul/modules/install-consul/install-consul --version {{user `consul_version`}}",
+      "/tmp/terraform-aws-consul/modules/install-consul/install-consul --version {{user `consul_version`}}"
+    ],
+    "pause_before": "30s"
+  },{
+    "type": "shell",
+    "only": ["ubuntu16-ami", "amazon-linux-2-ami"],
```

Comment:
I suspect that the code real users will want in their Packer templates will _not_ have all the different Linux distros. Perhaps in the README, we only show one of the distros, and add a comment that you may need to swap dnsmasq or systemd-resolved depending on the distro you're using?

Reply:
I think this gives more clarity and there is no harm and keeping it clear but happy to change if you think otherwise.